src/components/Card/Card.jsx

```diff
@@ -36,8 +40,8 @@ const Card = () => {
     }
 
     if (login === "username" && password === "password") {
-      localStorage.setItem('authToken', token);
-      console.log('Success');
+      localStorage.setItem('token', '123456');
+      navigate("/products");
```

Comment:
create constants folder and routes.js file to keep routes - https://www.albertgao.xyz/2019/09/07/reuse-react-router-routes-constants-with-hooks-when-navigating/